File: flutter/lib/modelo/Exercicio.dart
class Exercicio {
  final String nome;
  final String descricao;
  final int series;
  final int repeticoes;
  final String equipamento;

  Exercicio(
      this.nome,
      this.descricao,
      this.series,
      this.repeticoes,
      this.equipamento,
      );

  @override
  String toString() {
    // TODO: implement toString
    return 'Pessoa{$nome, $descricao, $series, $repeticoes, $equipamento}';
  }

  Exercicio.fromJson(Map<String, dynamic> json)
      : nome = json['nome'],
        descricao = json['descricao'],
        series = json['series'],
        repeticoes = json['repeticoes'],
        equipamento = json['equipamento'];

  Map<String, dynamic> toJson() => {
    "nome": nome,
    "descricao": descricao,
    "series": series,
    "repeticoes": repeticoes,
    "equipamento": equipamento
  };
}

File: springboot/src/main/java/com/academia/entity/Exercicio.java
package com.academia.entity;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Exercicio {
	
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id_exercicio")
    private long id;
    
    @Column
    private String nome;
    @Column
    private String descricao;
    @Column
    private Integer series;
    @Column
    private Integer repeticoes;
    @Column
    private String equipamento;
    
	public long getId() {
		return id;
	}
	public void setId(long id) {
		this.id = id;
	}
	public String getNome() {
		return nome;
	}
	public void setNome(String nome) {
		this.nome = nome;
	}
	public String getDescricao() {
		return descricao;
	}
	public void setDescricao(String descricao) {
		this.descricao = descricao;
	}
	public Integer getSeries() {
		return series;
	}
	public void setSeries(Integer series) {
		this.series = series;
	}
	public Integer getRepeticoes() {
		return repeticoes;
	}
	public void setRepeticoes(Integer repeticoes) {
		this.repeticoes = repeticoes;
	}
	public String getEquipamento() {
		return equipamento;
	}
	public void setEquipamento(String equipamento) {
		this.equipamento = equipamento;
	}

    
}